osu.Game/Rulesets/Mods/ModEasy.cs

```diff
@@ -26,5 +28,22 @@ public void ApplyToDifficulty(BeatmapDifficulty difficulty)
             difficulty.DrainRate *= ratio;
             difficulty.OverallDifficulty *= ratio;
         }
+
+        public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
+        {
+            //Note : The lives has to be instaciated here in order to prevent the values from different plays to interfear
+            //with each other / not reseting after a restart , as this method is called once a play starts (to my knowlegde).
+            //This will be better implemented with a List<double> once I know how to reliably get the game time and update it.
+            //If you know any information about that, please contact me because I didn't find a sollution to that.
+            lives = 2;
+            scoreProcessor.Health.ValueChanged += valueChanged =>
+            {
+                if (scoreProcessor.Health.Value == scoreProcessor.Health.MinValue && lives > 0)
+                {
+                    lives--;
```

Comment:
You'll want to make an interface, `IUseGameplayClock` or similar, and implement it here. `GameplayClockContainer` will then do something like:

```
foreach (var m in beatmap.Mods.Value.OfType<IUseGameplayClock>())
    m.ApplyGameplayClock(GameplayClock);
```